components/molecule/checkboxField/src/index.js

```diff
@@ -18,6 +18,8 @@ const MoleculeCheckboxField = ({
   errorText,
   helpText,
   onChange,
+  auxIcon: AuxIcon,
```

Comment:
Seeing what he receives maybe for more semantics you could use `toggleItemIcon: ToggleItemIcon,`